components/ImprovedSearch/PaginationItem.tsx

```diff
@@ -0,0 +1,93 @@
+import { usePagination } from 'react-instantsearch-hooks-web'
+import { Button } from "./Button"
+import styled from 'styled-components'
+
+
+const StyledListItem = styled.li`
+  display: inline-block;
+`
+
+const PaginationLink = styled(Button)<{ isCurrent?: boolean; inverted?: boolean }>`
+  width: 44px;
```

Comment:
Might consider a CSS variable here, as it seems like this should be a circle. Will make it easier to change size later on, if needed

Reply:
I think I would probably done that yes. :) Like
```
--button-size: 44px;
width: var(--button-size);
height: var(--button-size);
```
But in our case here, we could probably also use the round variant of EDS? Don't they have something like that? I guess 44px is something that is needed in Envis, and not necessarily something we will need